### DISCUSSION TITLE
PPINK : flagged as Suspicious during sandbox analysis

### DISCUSSION THREAD

**Discussion by mohit0121** [pubpub-zz/ppInk#289] [Q&A]
Hello Team, we are trying to whitelist this app but we have encountered these issues by our Infosec team.

1. Why this app is not digitally signed by a  trusted CA?

2. This app is flagged as Suspicious during sandbox analysis. Attached report for reference from Falcon CrowdStrike

[ppInk.exe _ Sandbox _ Counter Adversary Operations _ Dynamic Analysis.pdf](https://github.com/pubpub-zz/ppInk/files/13938069/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Dynamic.Analysis.pdf)
[ppInk.exe _ Sandbox _ Counter Adversary Operations _ Intelligence.pdf](https://github.com/pubpub-zz/ppInk/files/13938071/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Intelligence.pdf)
[ppInk.exe _ Sandbox _ Counter Adversary Operations _ Mitre Attack.pdf](https://github.com/pubpub-zz/ppInk/files/13938072/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Mitre.Attack.pdf)
[ppInk.exe _ Sandbox _ Counter Adversary Operations _Static Analysis.pdf](https://github.com/pubpub-zz/ppInk/files/13938073/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._Static.Analysis.pdf)

3. Virus total >> https://www.virustotal.com/gui/file/73fe4fef701bf731274e6e7efd97a1a91566e842ba44f70230fb81e433240736/details

4. Are these detections false positive? if yes, can they be safely ignored?

Request your Kind confirmation.

**Comment by pubpub-zz** [comment]
> Hello Team, we are trying to whitelist this app but we have encountered these issues by our Infosec team.

sorry Not a team (unfortunately). I'm just on my own.

> 
>     1. Why this app is not digitally signed by a  trusted CA?

Application is released as an open source therefore any one can recompile and sign the application.
I currently don't have any trusted certificate to use. if you can propose me with a solution to get some I may think about.

> 
>     2. This app is flagged as Suspicious during sandbox analysis. Attached report for reference from Falcon CrowdStrike
> 
> 
> [ppInk.exe _ Sandbox _ Counter Adversary Operations _ Dynamic Analysis.pdf](https://github.com/pubpub-zz/ppInk/files/13938069/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Dynamic.Analysis.pdf) [ppInk.exe _ Sandbox _ Counter Adversary Operations _ Intelligence.pdf](https://github.com/pubpub-zz/ppInk/files/13938071/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Intelligence.pdf) [ppInk.exe _ Sandbox _ Counter Adversary Operations _ Mitre Attack.pdf](https://github.com/pubpub-zz/ppInk/files/13938072/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Mitre.Attack.pdf) [ppInk.exe _ Sandbox _ Counter Adversary Operations _Static Analysis.pdf](https://github.com/pubpub-zz/ppInk/files/13938073/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._Static.Analysis.pdf)

I have not been able to look through all elements but at least I can say:
"Malicious : Contains ability to capture the screen" -> this is the main function of this application 😉
"Suspicious : Found a potential E-Mail address in binary/memory" -> this is the mail address for @geovens. I keep it in order to respect copyright.
"Suspicious: Contains ability to retrieve keyboard strokes" -> required to get the global shortcut to activate the application
"Suspicious: Contains ability to open a port and listen for incoming connection" -> this is to be able to provide REST API control capability
(sorry for the other entries I've not been through)

> 
>     3. Virus total >> https://www.virustotal.com/gui/file/73fe4fef701bf731274e6e7efd97a1a91566e842ba44f70230fb81e433240736/details
>     4. Are these detections false positive? if yes, can they be safely ignored?

The detection looks like a false positive : you should be able to report it for deeper analysis.

> 
> 
> Request your Kind confirmation.

Hoping this will help you and feel free to star the project.

**Comment by mohit0121** [comment]
@pubpub-zz  Thank you for taking out time in reviewing few of the detections., Much appreciated !

Would you be kind enough in reviewing the  attached document ( 23 Detections) and provide your inputs, detection by detection.  This will help us to whitelist the application and this tool will be deployed and used by our company globally. doable?

[ppInk.exe _ Sandbox _ Counter Adversary Operations _ Dynamic Analysis.pdf](https://github.com/pubpub-zz/ppInk/files/13963209/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Dynamic.Analysis.pdf)

**Comment by pubpub-zz** [comment]
I've added my comments.
[ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Dynamic.Analysis-1rep.pdf](https://github.com/pubpub-zz/ppInk/files/13967543/ppInk.exe._.Sandbox._.Counter.Adversary.Operations._.Dynamic.Analysis-1rep.pdf)

If you finally deploy ppInk in your company, : 
* remember to respect the copyright and the license
* I would be pleased if you star the project and recommend other users to do so
* finally, my software is open source and free, however to support my work, I would appreciate if your company could do a donation to support girls' education (unicef or any one else) ☺️

**Comment by mohit0121** [comment]
Certainly, Please guide me how to go about starring the project?

**Comment by pubpub-zz** [comment]
> Certainly, Please guide me how to go about starring the project?

![image](https://github.com/pubpub-zz/ppInk/assets/4083478/fd22112e-c551-41c1-9f02-8102a2467f2b)
